src/main/resources/templates/script.js

```diff
@@ -34,4 +34,11 @@ socket.onmessage = function(event){
             console.log(data[key])
         }
     }
-}
\ No newline at end of file
+}
+
```

Comment:
moved the internal script code to existing external JS file, this increases efficiency and reduces technical debt.

ref: https://martinfowler.com/bliki/TechnicalDebt.html